ivy/functional/ivy/general.py

```diff
@@ -1317,9 +1317,62 @@ def value_is_nan(
     ret
         Boolean as to whether the input value is a nan or not.
 
+    Examples
+    --------
+    >>> x = ivy.array([451])
+    >>> y = x.value_is_nan()
+    >>> print(y)
+    False
+
+    >>> x = ivy.array([float('inf')])
+    >>> y = x.value_is_nan()
+    >>> print(y)
+    True
+
+    >>> x = ivy.array([float('inf')])
+    >>> y = x.value_is_nan(include_infs=False)
+    >>> print(y)
+    False
+
+    >>> x = ivy.array([float('nan')])
+    >>> y = x.value_is_nan(include_infs=False)
+    >>> print(y)
+    True
+
+    >>> x = ivy.array([0])
+    >>> y = x.value_is_nan()
+    >>> print(y)
+    False
+
+    With :code:`ivy.Container` input:
+    >>> x = ivy.Container(a=ivy.array([420.]), b=ivy.array([72]))
+    >>> y = ivy.value_is_nan(x)
+    >>> print(y)
+    {
+        a: False,
+        b: False
+    }
+
+    With one :code:`ivy.Container` static method:
```

Comment:
the examples starting from here are not needed as they are a repeat of the ones in `container/general.py`